### ISSUE TITLE
Akitio Node/Radeon does not detect display

### ISSUE THREAD

**Issue by mixxit** [clearlinux/distribution#996]
I have an XPS 13 9360 which has no hdmi/output etc. I can connect either an Akitio Node with a radeon inside or a thunderbolt HP dock which has DisplayLink adapters inside it

Sadly displaylink does not seem to work so I cannot use my HP Dock

And the akitio node is detected by thunderbolt in settings and i can authorize it but no displays are detected

So when i am sat at my desk i have to stare at my tiny laptop screen

**Comment by ahkok** [comment]
Likely not a bug but a feature we have not enabled. Without us actually having this hardware, it will be difficult to make this work. Since we don't know what feature is missing, we need more information.

**Comment by mixxit** [comment]
sure what do you need me to provide?

**Comment by mrkz** [comment]
@ahkok after some chatting with @mixxit on IRC turns out BIOS setting was really restrictive for Thunderbolt devices, after changing BIOS setting, screens could be accessed and used.

Those only work on Wayland though (xrandr --listmonitors only shows 1 monitor [built-in display])

**Comment by mixxit** [comment]
Hi @ahkok 

With the help of @mrkz  and doctorhu[m] in the IRC channel they informed me that with EGPU like the Akitio node it might be necessary to make some BIOS changes for Thunderbolt Security

I currently have an XPS 13 9360 and the BIOS settings where:

![image](https://user-images.githubusercontent.com/233287/61141736-2273b900-a4c6-11e9-99c3-2db31065e738.png)

So i changed it to 

![image](https://user-images.githubusercontent.com/233287/61141759-2e5f7b00-a4c6-11e9-9a6e-fa0f86439aa1.png)


So i made these changes and booted back into Clearlinux and to my suprise on the login screen all of my monitors activated!

However, after logging in, the monitors disabled themselves and it went back to my laptop monitor

After further discussion with @mrkz he requested I try wayland on the login screen

So i selected GNOME and it went to my desktop and all screens were working

![image](https://user-images.githubusercontent.com/233287/61141863-6c5c9f00-a4c6-11e9-9c15-f62dceaf5fed.png)

As requested I have also attached my Xorg.0.log file

[Xorg.0.log](https://github.com/clearlinux/distribution/files/3387168/Xorg.0.log)

I hope this helps solve the issue

**Comment by ahkok** [comment]
Can we close this issue then?

**Comment by mixxit** [comment]
I guess whilever wayland is not the default desktop then the bug is still going to be there for anyone with this configuration (Xorg)

I dont know what Ubuntu do but i think they dont use wayland as default? The issue does not occur there so maybe they have some setting that would help fix it?

Thanks

**Comment by mrkz** [comment]
@ahkok I think there's an issue within Xorg when using this setup, so real bug (as using a Wayland session is just a workaround).

Having said that, It would be a bit tricky/hard to follow-up given the lack of hardware.

**Comment by mixxit** [comment]
Does Ubuntu use Xorg by default? As it works perfectly fine there so Im not sure whatever configuration they have clearlinux should probably use?
